meteor/server/api/rundownLayouts.ts

```diff
@@ -88,6 +80,39 @@ PickerPOST.route('/shelfLayouts/upload/:showStyleBaseId', async (params, req: In
 	res.end(content)
 })
 
+/**
+ * Uploads the layout to the first Show Style Base that uses the given blueprint id
+ */
+PickerPOST.route(
+	'/shelfLayouts/uploadByBlueprintId/:showStyleBlueprintId',
+	async (params, req: IncomingMessage, res: ServerResponse) => {
```

Comment:
What is the type of params?

Reply:
`Params`, which is inferred, and it's okay.

Reply:
I disagree with it being okay, but not enough that you should change it in this PR.

There are two reasons for why we add type definitions:
- So that our code is strictly typed (this is the most important one)
- So that a maintenance developer (or reviewer) can easily and quickly understand what is going on in the code and know what a given variable contains.

When a type is inferred only the first reason is sastified, but a maintenance developer will have to spend extra time to figure out the type of `params`. Right here is not that bad, but there are places in Sofie where you have to go 3 or more layers "up the chain" to find the type definition and that is an unecessary burden. Also, a reviewer of the code will most likely review the code in a format that doesn't allow to navigate the code (for example right here on GitHub). 

It seems trivial/not important to add type definitions, but it's all these little things that takes a codebase from being good, to being great.

In this example `params` is inferred to be `Params`, but it might as well have been inferred to be `ParamsWithSomeExtraAttributeNoOneCanGuess extends Params`.